### DISCUSSION TITLE
Will knowledge center be available on the Open Source version?

### DISCUSSION THREAD

**Discussion by leovrf** [open-metadata/OpenMetadata#14689] [Q&A]
I've noticed the knowledge center release mentions "Collate" only hence why my question.

Several companies create their own "Data Portal" i.e. a single place with all data-related information available usually on top of open source markdown documentation tools such as [MKDocs](https://www.mkdocs.org/), [Docusaurus](https://docusaurus.io/) or even [Backstage](https://backstage.io/docs/overview/what-is-backstage).

Problem is all these tools do not seamlessly connect with the data ecosystem like Open Metadata and the knowledge center feature would be a perfect replacement.

**Question**
* Will knowledge center be available on the Open Source version of Open Metadata?

**Comment by betterzjy** [comment]
good question. also concerned about that

**Comment by harshach** [comment]
hi @leovrf @betterzjy knowledge center is currently Collate SaaS only.
@betterzjy could you please provide what concerns you have.

We are pushing tremendous amount of features into Open Source. That will continue to be the case

**Reply by betterzjy** [reply]
thanks for reply. we are building a data portal for a chinese company based on the openmetadata open-source version, it will be very helpful if you can add knowledge center feature to the open source version. 
best regards !

**Reply by betterzjy** [reply]
@harshach
